### DISCUSSION TITLE
AOT release build doesn't compile

### DISCUSSION THREAD

**Discussion by assassin316** [unoplatform/uno#11450] [General]
Following the instructions below (using latest Uno packages) results in a compilation error:

https://platform.uno/blog/optimizing-uno-platform-webassembly-applications-for-peak-performance/

![image](https://user-images.githubusercontent.com/17514380/221083830-be0811ad-c0e3-48c7-8544-f9c94b615166.png)

My guess is that I'm missing a step to perform?

**Comment by jeromelaban** [comment]
Thanks for the report! You may find more information about this by [looking at the binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html).

**Comment by assassin316** [comment]
Going back to this issue, my Wasm project head hsa the default settings in place, and compiling in Release gives the following error:

`System.Exception: Failed to execute the linker`

Looking at the binlog, doesn't give much information:

![image](https://github.com/unoplatform/uno/assets/17514380/0ac215d6-6650-4fc0-974e-a6ddd1bbd1d4)

Is there any other errors I can dig into to investigate further?  With default settings, do I need to have the WSL installed or is that only needed for full/profile-guided aot releases?

Thanks.

**Reply by jeromelaban** [reply]
The binlog will provide more information when the logging is set to diagnostics, for the task that shows that particular error. What do you see there?
